toolz/itertoolz.py

```diff
@@ -8,6 +8,13 @@
                                  filter)
 from toolz.utils import no_default
 
+try:
```

Comment:
It seems that this sort of code lives in compatibility.py.

Reply:
Thanks, updated!